graph/src/schema/api.rs

```diff
@@ -624,6 +624,7 @@ fn id_type_as_scalar(
     }?;
     let scalar_type = id_type.map(|id_type| match id_type {
         IdType::String | IdType::Bytes => ScalarType::new(String::from("String")),
+        IdType::Int8 => ScalarType::new(String::from("Int")),
```

Comment:
Shouldn't this be `"Int8"`?

Reply:
Nice catch, and a fun way to spend a few hours why this works with `Int` but not with `Int8` - the upshot is that `Int8` gets treated as a string internally in too many places, and queries just blow up in that case. Addressing that is a little scary and I've opened the separate PR #5034 to do that.

For this PR, I just left a big comment why it's `Int` and not `Int8` for now.